client/src/components/History/HistoryViews/JobHistoryModal.jsx

```diff
@@ -145,18 +156,25 @@ function JobHistoryContent({ jobDetails, jobDisplay, jobName, toggle }) {
       <div className={`${styles['left-panel']} ${styles['panel-content']}`}>
         <DescriptionList
           density="compact"
-          data={{
-            Output: !hideDataFiles && (
-              <DataFilesLink
-                path={outputLocation}
-                disabled={!isOutputState(jobDetails.status)}
-              >
-                View in Data Files
-              </DataFilesLink>
-            ),
-          }}
+          data={
+            !hideDataFiles && {
+              Execution: (
+                <DataFilesLink
+                  path={getExecutionPath(jobDetails)}
+                  disabled={hasOutput}
+                >
+                  View in Data Files
+                </DataFilesLink>
+              ),
+              Output: (
+                <DataFilesLink path={outputLocation} disabled={!hasOutput}>
```

Comment:
For my failed job (due to staging issues to do the temporary v3-ssh issue), the link is pointing to https://cep.test/workbench/data/tapis/private/ and is enabled.  Failed jobs typically have output but there one doesn't as it failed so early, right?  Are there any more things we can do to determine there is output? Maybe a follow on ticket?

Reply:
Yeah, not too sure honestly. I asked [CIC](https://tacc-team.slack.com/archives/C0307MGPCP2/p1674249846283739) about this, and they mentioned 

> If a job (status == FINISHED) or (status == FAILED && remoteOutcome != FAILED_SKIP_ARCHIVE), then the output is in the archive directory of the archive system.  Otherwise, the output is left in the execSystemOutputDir of the execution system.

Maybe we should also check that `remoteOutcome` is not blank? What's the value of `remoteOutcome` for your failed job with no actual output folder?

Reply:
>Maybe we should also check that remoteOutcome is not blank? What's the value of remoteOutcome for your failed job with no actual output folder?

sounds good. `remoteOutcome` is null.

![Screenshot 2023-02-15 at 4 52 59 PM](https://user-images.githubusercontent.com/8287580/219207065-2ec92cab-af67-4109-911f-66684cb3a574.png)
![Screenshot 2023-02-15 at 4 52 47 PM](https://user-images.githubusercontent.com/8287580/219207068-3233af5f-9f11-45bf-bfe5-b42cf764399c.png)